src/shared/components/FormComponents/TagsController/TagsController.styles.ts

```diff
@@ -14,22 +14,24 @@ export const StyledTextField = styled(TextField, shouldForwardProp)`
     }
   }
 
-  ${({
-    isSecondaryUiType,
-    showInputLabel,
-    setMinWidth,
-  }: {
-    isSecondaryUiType: boolean;
-    showInputLabel: boolean;
-    setMinWidth: boolean;
-  }) =>
+  input::placeholder {
+    color: ${variables.palette.outline};
+    opacity: 1;
+  }
+
+  ${({ isSecondaryUiType, setMinWidth }: { isSecondaryUiType: boolean; setMinWidth: boolean }) =>
     isSecondaryUiType &&
     `
     .MuiInputBase-root {
       min-height: 5.6rem;
-      padding: ${theme.spacing(0.4, 1, 0.4, 4)};
```

Comment:
Because of these changes showing emails in two rows will be with overlapping borders of input:
![image](https://github.com/user-attachments/assets/9ab92df4-045d-4b3d-8a74-876b8ee99238)